.github/workflows/format.yml

```diff
@@ -9,16 +9,12 @@ jobs:
     steps:
     - uses: actions/checkout@v3
 
-    - name: Set up Python
-      uses: actions/setup-python@v4
+    - name: Install uv
+      uses: astral-sh/setup-uv@v5
       with:
-        python-version: 3.11
-
-    - name: Install black formatter
-      run: |
-        python -m pip install --upgrade pip
-        pip install black==22.12.0
+        version: "0.5.29"
+        python-version: "3.12"
 
     - name: Check formatting
       run: |
-        python -m black -l 100 mrmustard/ tests/ --check
+        uv run --only-dev black -l 100 mrmustard/ tests/ --check
```

Comment:
This `-l 100` likely needs to be passed as an arg in the `.pre-commit-config.yaml`, or removed here, yeah?